src/Core/Models/Api/Request/PushDeviceRequestModel.cs

```diff
@@ -7,6 +7,5 @@ public class PushDeviceRequestModel
 {
     [Required]
     public string Id { get; set; }
-    [Required]
     public DeviceType Type { get; set; }
```

Comment:
Is this property needed at all anymore?

Reply:
No and now that you mention it, there's no reason not to remove, since the deserialization works either way.